src/grid/Col.js

```diff
@@ -1,8 +1,13 @@
+/*eslint-disable no-console */
 import PropTypes from 'prop-types';
 import React from 'react';
 import { View, StyleSheet, TouchableOpacity } from 'react-native';
 
 const Col = props => {
+  console.warn(
+    'Warning: Col has been deprecated and will be removed in a future version of React Native Elements'
```

Comment:
Are we separating out the grid into a separate package under react-native-training? @Monte9